app/client/oepn_ai.py

```
@@ -0,0 +1,15 @@
+
+from openai import OpenAI
+
+_gpt_client: OpenAI | None = None
+
+def get_gpt_client() -> OpenAI:
+    """OpenAI 클라이언트를 싱글톤으로 제공한다."""
+    global _gpt_client
+    if _gpt_client is None:
+        api_key = os.getenv("OPENAI_API_KEY")
+        if not api_key:
+            raise RuntimeError("OPENAI_API_KEY 환경변수가 설정되어 있지 않습니다.")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)